app/models/ministry.rb

```diff
@@ -85,7 +85,8 @@ def team_members # rubocop:disable Metrics/AbcSize
               !members[assignment.person_id].try(:inherited_role?)
 
       # Keep highest Inherited assignment per person
-      if members[assignment.person_id].blank? || members[assignment.person_id].try(:[], :role) < assignment[:role]
+      if members[assignment.person_id].blank? ||
+         Assignment.roles[members[assignment.person_id].try(:[], :role)] < Assignment.roles[assignment[:role]]
```

Comment:
Many areas like this.  enum fields now return the string instead of the integer so I had to cast it into an integer like this.